docs/src/osdk/reference/environment-variables.md

```diff
@@ -0,0 +1,18 @@
+# Environment Variables
+
+The OSDK tool uses the following environment variables to customize compilation and target behavior. 
+
+## OSDK-Specific Variables
+
+### `INTEL_TDX`
+Enables Intel Trusted Domain Extensions (TDX) support.  
+**Values:**
+- `1` or `true`: Enables TDX-specific compilation
+- Unset: Disables TDX features  
+**Requires:** Compatible Intel CPU and toolchain
+
+### `OSDK_TARGET_ARCH`
+Overrides default target architecture.  
+**Values:**
+- Valid value: `x86_64`, `riscv64`, `loongarch64`  
+- Unset: Uses host architecture as default 
```

Comment:
Follow [the writing style and format of the Cargo book](https://doc.rust-lang.org/cargo/reference/environment-variables.html). For example, each environment variable should be given a bullet point. And the doc should be written in complete sentences. I believe I have explained this point to you multiple times: documentation are not slides. So do not write documentation as if you are writing slides.